### PR TITLE
Convert `lib/sha1` to ES6

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Converts `lib/sha1` to ES6.

## What is the value of this and can you measure success?

ES6! It was fun to convert.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Tested in CODE?

No.

### CODE REVIEW

**Comment by sndrs** [comment]
is this something we should install?

**Comment by gustavpursche** [comment]
I thought the same, but wasn't able to find any dependency-free trustworthy source for it. I guess that's the reason why this was added initially ...

File: static/src/javascripts/lib/sha1.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import sha1 from './sha1';
+
+const TABLE = [
+    ['A cryptographic hash', 'de6094f2e556845db1529b5f42c72b2985bf6393'],
+    ['integrity verification', 'b8ff2e013132a20fc79b145d65377f5325c500ff'],
+    ['password storage', '7495b1e3fe2662663e5557790ec33c227db2386d'],
```
Comment (SiAdcock) [inline static/src/javascripts/lib/sha1.spec.js]: Were these hashes generated by `lib/sha1.js` or something else?

Comment (gustavpursche) [inline static/src/javascripts/lib/sha1.spec.js]: By something else :)

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**